concourse/steps/component_descriptor.mako

```diff
@@ -80,6 +83,34 @@ component_labels = ${descriptor_trait.component_labels()}
 component_name_v2 = component_name.lower() # OCI demands lowercase
 ctx_repository_base_url = '${descriptor_trait.ctx_repository_base_url()}'
 
+ocm_repository_mappings = ${ocm_repository_mappings}
+
+if ocm_repository_mappings:
+    ocm_repository_mappings = [
+        dacite.from_dict(
+            data_class=cnudie.util.OcmLookupMapping,
+            data=e,
+        ) for e in ocm_repository_mappings
+    ]
+else:
+    ocm_repository_mappings = [
+        dacite.from_dict(
+            data_class=cnudie.util.OcmLookupMapping,
+            config=dacite.Config(
+                cast=[
+                    oa.Privileges,
+                ],
+            ),
+            data={
+                'ocm_repo_url': ctx_repository_base_url,
+                'component_names': ['.*',],
```

Comment:
does OCM-Cli also specify regex? also: did they not call this (component-)prefix? I would prefer to keep as close as reasonably possible to their [spec](https://github.com/open-component-model/ocm/blob/main/docs/reference/ocm_configfile.md)

Reply:
@AndreasBurger : yes, I think we should keep close to OCM-cli to
a) make migration easier (should we ever need to)
b) offer a somewhat consistent interface at least